docs/BREAKINGCHANGES.md

```diff
@@ -163,7 +139,7 @@ TODO
 
 
 ### Make error message consistent when invalid script is passed to -File, better error when passed ambiguous arg [4573](https://github.com/PowerShell/PowerShell/issues/4573)
-TODO
+TODO (I believe the breaking change here is for "Enable -WindowStyle to work", changing the exit code.  Need to confirm)
```

Comment:
No need to mention enabling -WindowStyle to work, the breaking change is modifying exit codes of pwsh.exe to align with Unix conventions.